miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -1403,7 +1403,7 @@ public static LibraryDto asDto(@Nonnull Library from, boolean includeBoxPosition
     dto.setParentSampleAlias(from.getSample().getAlias());
     dto.setProjectId(from.getSample().getProject().getId());
     setString(dto::setProjectName, from.getSample().getProject().getName());
-    setString(dto::setProjectShortName, from.getSample().getProject().getShortName());
```

Comment:
Please modify `LibraryDto` to change `get/setProjectShortName` -> `get/setProjectCode`